diff --git a/dist/core.esm.js b/dist/core.esm.js
index a2ea504093367cf66be3c0149bd9d27536b9f492..42a4fb781d47eaa06542a600058f67b1458d3a06 100644
--- a/dist/core.esm.js
+++ b/dist/core.esm.js
@@ -1440,7 +1440,8 @@ class AbstractPointerSensor {
       this.listeners.add(events.cancel.name, this.handleCancel);
     }
 
-    this.windowListeners.add(EventName.Resize, this.handleCancel);
+    // !NOTE: Commenting this will make dnd work in Firefox addon popup
+    // this.windowListeners.add(EventName.Resize, this.handleCancel);
     this.windowListeners.add(EventName.DragStart, preventDefault);
     this.windowListeners.add(EventName.VisibilityChange, this.handleCancel);
     this.windowListeners.add(EventName.ContextMenu, preventDefault);
