diff --git a/dist/core.esm.js b/dist/core.esm.js
index 1577c253774e3c37eafa4f267f1944426deee405..e3be5438693269cba18a7062cdb59c5a6210a166 100644
--- a/dist/core.esm.js
+++ b/dist/core.esm.js
@@ -1441,7 +1441,8 @@ class AbstractPointerSensor {
       this.listeners.add(events.cancel.name, this.handleCancel);
     }
 
-    this.windowListeners.add(EventName.Resize, this.handleCancel);
+    // !NOTE: Commenting this will make dnd work in Firefox addon popup
+    // this.windowListeners.add(EventName.Resize, this.handleCancel);
     this.windowListeners.add(EventName.DragStart, preventDefault);
     this.windowListeners.add(EventName.VisibilityChange, this.handleCancel);
     this.windowListeners.add(EventName.ContextMenu, preventDefault);
